spec/pg_dump_anonymize_spec.rb

```diff
@@ -3,7 +3,62 @@
     expect(PgDumpAnonymize::VERSION).not_to be nil
   end
 
-  it "does something useful" do
-    expect(false).to eq(true)
+  it "can process a file with no transformations, producing an exact copy" do
```

Comment:
didn't put too much thought into these, definitely not suggesting they come close to exhausting the set of behaviour that would be good to test